astropy/time/tests/test_basic.py

```diff
@@ -1056,6 +1056,9 @@ def test_to_datetime():
         assert tz.tzname(dt) == tz_dt.tzname()
     assert np.all(time == forced_to_astropy_time)
 
+    with pytest.raises(ValueError):
+        Time('2015-06-30 23:59:60.000').to_datetime()
```

Comment:
You could copy & paste the message you get from an interactive test run, or just check part of the message, as in
```
assert 'datetime does not support' in str(message)
```
I personally prefer the latter -- I don't think it is all that important to ensure a test message is always identical, we just want to ensure we get the one that you put it!